bool isActive
bool isComplete
int32 workerID
string type
int32 zone
Point start_point
---
